ecoli/experiments/ecoli_master_sim.py

```diff
@@ -542,16 +506,20 @@ def run(self):
             # with suffix_time = True.
             if not self.experiment_id_base:
                 self.experiment_id_base = self.experiment_id
-
             if self.suffix_time:
-                self.experiment_id = datetime.now().strftime(f"{self.experiment_id_base}_%d/%m/%Y %H:%M:%S")
+                self.experiment_id = datetime.now().strftime(
+                    f"{self.experiment_id_base}_%d/%m/%Y %H:%M:%S")
             experiment_config['experiment_id'] = self.experiment_id
         experiment_config['profile'] = self.profile
 
+        # Since unique numpy updater is an class method, internal
+        # deepcopying in vivarium-core causes this warning to appear
+        warnings.filterwarnings("ignore",
```

Comment:
`vivarium-core` deep-copies ports schemas when configuring stores. This causes a warning to pop up because each of the unique molecule updaters is a method bound to a specific instance of a class. Not sure if there is a more elegant way to handle this.

Reply:
Just double checking -- the updater is a class instance because it requires the class's indices instances? That makes sense, and no way to deep copy that. I don't remember why that deep copy was implemented, but as long as the updater is not changing during the simulation I think it should be okay. Using this warnings filter is not great, but better than continuing to get the warning. If you think something should be added to vivarium-core, I could support that.

Reply:
I’ve quoted the updater in its entirety in the OP, but the important instance variables are used to accumulate unique molecule updates until the time step has ended, at which point they are sorted before being applied.